bot/resources/tags/regex.md

```diff
@@ -0,0 +1,15 @@
+**Regular expressions**
+Regular expressions (regex) are a tool for finding patterns in strings. The standard library's `re` module defines functions for using regex patterns.
+
+**Example**
+We can use regex to pull out all the numbers in a sentence:
+```py
+>>> import re
+>>> x = "On Oct 18 1963 a cat was launched aboard rocket #47"
+>>> regex_pattern = r"[0-9]{1,3}"  # Matches one-three digits
+>>> re.findall(regex_pattern, foo)
+['18', '196', '3', '47']         # Notice the year is cut off
```

Comment:
This comment is a bit too far to the right. If there's two spaces instead of this many, then it can all fit on one line, making the embed smaller. Oberve.

With many spaces:
![image](https://user-images.githubusercontent.com/15021300/158041877-39a8d46f-6591-4ec8-acd4-2817f1b6824d.png)

With two spaces:
![image](https://user-images.githubusercontent.com/15021300/158041889-b23e0e18-b6ec-4fe8-ba18-44c553f4f4a1.png)

```suggestion
['18', '196', '3', '47']  # Notice the year is cut off
```

Reply:
I also shortened the other comment